cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -354,6 +354,91 @@ objects:
         "cloudEnvironment": "AzurePublicCloud"
       }
 
+# TODO: parameterize data.platform-identities.yaml after templatize refactor and helm chart conversion
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: clusters-service-platform-identity-roleset
```

Comment:
Can we align the naming to the one used by CS;
We are currently using `azure-operators-managed-identities-config`

Reply:
The reasoning for this suggestion is that it'll be easier for devs (notably CS ones) to work with this configMap if it contains the same naming.